apps/linode-marketplace-zabbix/provision.yml

```diff
@@ -0,0 +1,26 @@
+# zabbix
+- name: setting up environment 
+  hosts: localhost
+  connection: local
+  any_errors_fatal: true
+  user: root
+  vars_files:
+    - group_vars/linode/vars
+  tasks:
+    - name: generating sudo user and password 
+      import_tasks: ../linode_helpers/roles/sudouser/tasks/main.yml
+  
+    - name: generating MySQL password
+      import_tasks: ../linode_helpers/roles/securemysql/tasks/passgen.yml
+
+    - name: generating zabbix password
+      set_fact:
+        zabbix_db_pass: "{{ lookup('password', '/dev/null length=45 chars=ascii_letters,digits') }}"
+      no_log: true
+
+    - name: writing sudo user password to vars
+      lineinfile:
+        insertafter: EOF
+        path: group_vars/linode/vars
+        line: | 
+          zabbix_db_pass: {{ zabbix_db_pass }}
```

Comment:
```suggestion
    - name: writing zabbix database password to vars
      lineinfile:
        insertafter: EOF
        path: group_vars/linode/vars
        line: | 
          zabbix_db_pass: {{ lookup('password', '/dev/null length=45 chars=ascii_letters,digits') }}
```

Reply:
we don't have to set_fact to just write the creds to the vars file. We can just write directly.